releasenotes/notes/trace-propagation-style-extract-order-baa0637bc4f506f2.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Enforce ordering on ``DD_TRACE_PROPAGATION_STYLE_EXTRACT``` for feature parity across tracers. Previously the order of values provided did not matter, 
+    as the priority we put on the values was hard-coded: ``datadog, b3, b3 single header``. Now the extraction order respects the order provided for the value.
```

Comment:
```suggestion
    The specification of ``DD_TRACE_PROPAGATION_STYLE_EXTRACT`` requires the configured styles be evaluated in order. However, the evaluation order had been hard-coded and so ignored the configured ordering.
```

Reply:
We also should include an update note with any recommended changes for users that want the existing behavior, i.e., configure with the previous hard-coded ordering.